tt/problem_text.html.tt

```diff
@@ -233,10 +252,10 @@ $('.problem_text div.problem_menu li.langs a').click(function() {
       s.indexOf('pl=') < 0  ? s + ';pl=' + this.innerText : s.replace(/pl=[a-z]+/, 'pl=' + this.innerText);
   }
 });
-
 [%- IF href_get_last_verdicts -%]
 if (sid)
   $(document).ready(function () {
+    $('.submit-form').hide();
```

Comment:
This does not degrade nicely, logic should be inverted. See nearby code.
But also see above.